lib/blacklight/maps/maps_search_builder.rb

```diff
@@ -0,0 +1,20 @@
+module BlacklightMaps
+  module MapsSearchBuilder
```

Comment:
Hmm. That seems troubling. It's possible that the implementing application may already be using it's own SearchBuilder with custom logic (which is definitely the case with Digital Commonwealth), and we can't then force implementers to use BlacklightMaps::MapsSearchBuilder as ```config.search_builder_class```.

So if we're not injecting the logic via BlacklightMaps::ControllerOverride (https://github.com/projectblacklight/blacklight-maps/pull/74/files#diff-544618e3e5d4681c08c1bf5fc29e5584R10), then you'd need to be able to include BlacklightMaps::MapsSearchBuilder into a pre-existing SearchBuilder class in the implementing application. Which points more to a module, rather than a class, for BlacklightMaps::MapSearchBuilder?

The local implementor's SearchBuilder would look something like:

```ruby
class SearchBuilder < Blacklight::SearchBuilder
  include Blacklight::Solr::SearchBuilderBehavior
  include BlacklightMaps::MapsSearchBuilder

  def some_local_implementation_logic(solr_parameters = {})
    ...
  end

end
```